docs/characterization/xeb_coherent_noise.ipynb

```diff
@@ -0,0 +1,283 @@
+{
```

Comment:
<p>we're missing standard notebook headers</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3706/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
added